.github/workflows/build.yml

```diff
@@ -101,7 +91,7 @@ jobs:
           SLACK_COLOR: ${{env.SLACK_ERROR}}
           SLACK_MESSAGE: "There has been a failure building the application"
           SLACK_TITLE: "Failure Building Application"
-          SLACK_WEBHOOK: " ${{ steps.slack-web-hook.outputs.SLACK-WEBHOOK }} "
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
So for the build job should be able to remove the az login (L39-43) and the review environment (L27).